src/Components/OATPropertyEditor/FormAddEnumItem.tsx

```diff
@@ -191,6 +200,20 @@ export const FormAddEnumItem = ({
         return find ? `${t('OATPropertyEditor.errorRepeatedEnumValue')}` : '';
     };
 
+    const handleNameChange = (value) => {
+        const regularExpression = new RegExp(
+            '^[a-zA-Z](?:[a-zA-Z0-9_]*[a-zA-Z0-9])?$'
```

Comment:
I'd put a short explainer as a comment here so folks don't have to try and parse in their heads.